src/main/java/mate/academy/spring/validator/PasswordValidator.java

```diff
@@ -0,0 +1,31 @@
+package mate.academy.spring.validator;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import mate.academy.spring.dto.request.UserRequestDto;
+
+public class PasswordValidator implements ConstraintValidator<Password, UserRequestDto> {
+    private static final String PASSWORD_PATTERN = "^(?=.*[A-Za-z])(?=.*\\d).{4,}$";
+
+    @Override
+    public boolean isValid(UserRequestDto userRequestDto,
+                           ConstraintValidatorContext constraintValidatorContext) {
+        String password = userRequestDto.getPassword();
+        String repeatPassword = userRequestDto.getRepeatPassword();
+        if (userRequestDto != null && isMatchPattern(password)) {
```

Comment:
first you're getting repeatPassword and only after it checking if dto is null. Let's fix this error-prone code